shared/fs/row/still.js

```diff
@@ -1,152 +1,66 @@
+  type: Types.PathType,
 }
 
-const HoverBox = isMobile
-  ? Box
-  : glamorous(Box)({
-      '& .fs-path-item-hover-icon': {
-        color: globalColors.white,
-      },
-      ':hover .fs-path-item-hover-icon': {
-        color: globalColors.black_40,
-      },
-      '& .fs-path-item-hover-icon:hover': {
-        color: globalColors.black_60,
-      },
-    })
-
-const RowMeta = ({badgeCount, isDownloading, isNew, isIgnored, needsRekey}) => {
-  if (isIgnored || !(isDownloading || isNew || isIgnored || needsRekey || badgeCount)) {
+const RowMeta = ({isDownloading}) => {
+  if (!isDownloading) {
     return null
   }
 
   return (
     <Box style={{width: 0, display: 'flex'}}>
-      {needsRekey && (
-        <Box style={styleBadgeContainerRekey}>
-          <Meta title="rekey" backgroundColor={globalColors.red} />
-        </Box>
-      )}
-      {isNew && (
-        <Box style={styleBadgeContainerNew}>
-          <Meta title="new" backgroundColor={globalColors.orange} />
-        </Box>
-      )}
       {isDownloading && (
-        <Box style={styleDownloadContainer}>
+        <Box style={rowStyles.downloadContainer}>
           <Icon type="icon-addon-file-downloading" />
         </Box>
       )}
-      <Box style={styleBadgeContainer}>
-        {!!badgeCount && <Badge badgeNumber={badgeCount} badgeStyle={badgeStyleCount} />}
-      </Box>
     </Box>
   )
 }
 
 const Still = (props: StillProps) => (
-  <HoverBox style={rowStyles.rowBox}>
-    <ClickableBox onClick={props.onOpen} style={rowStyles.leftBox}>
-      <Box2 direction="vertical">
-        <PathItemIcon spec={props.itemStyles.iconSpec} style={rowStyles.pathItemIcon} />
-      </Box2>
-      <RowMeta badgeCount={props.badgeCount} {...props.tlfMeta} isDownloading={props.isDownloading} />
-      <Box style={rowStyles.itemBox}>
-        <Box2 direction="horizontal" fullWidth={true}>
-          <Text
-            type={props.itemStyles.textType}
-            style={rowStyles.rowText(props.itemStyles.textColor)}
-            lineClamp={isMobile ? 1 : undefined}
-          >
-            {props.name}
-          </Text>
-          {props.isEmpty && (
-            <Meta title="empty" backgroundColor={globalColors.grey} style={{marginLeft: globalMargins.tiny, marginTop: globalMargins.xxtiny}} />
-          )}
-        </Box2>
-        {props.type === 'folder' &&
-        (!props.resetParticipants || props.resetParticipants.length === 0) ? null : (
-          <PathItemInfo
-            lastModifiedTimestamp={props.lastModifiedTimestamp}
-            lastWriter={props.lastWriter}
-            resetParticipants={props.resetParticipants}
-            isUserReset={props.isUserReset}
+  <StillCommon
+    itemStyles={props.itemStyles}
+    name={props.name}
+    path={props.path}
+    onOpen={props.onOpen}
+    openInFileUI={props.openInFileUI}
+  >
+    <RowMeta isDownloading={props.isDownloading} />
+    <Box style={rowStyles.itemBox}>
+      <Box2 direction="horizontal" fullWidth={true}>
```

Comment:
Why are we using both `Box` and `Box2`? Can we just use one?

Reply:
I think we are migrating to Box2, but there are still a lot of Box left over. They still work so probably not worthwhile to do them together in this ticket?